app/views/state_file/questions/w2/edit.html.erb

```diff
@@ -46,13 +49,17 @@
       <% end %>
     </div>
 
-    <% if t(".box14_explanation.#{@state_code}_html", default: nil) %>
+    <% explanation_key = ".box14_explanation_#{@state_code}_html" %>
+    <% button_text = @state_code == "nj" ? t(".what_is_box14_nj") : t(".what_is_box14") %>
```

Comment:
only two states support box 14, is there a way to make this dynamic?
`    <% button_text =  t(".what_is_box14_#{state_code}") %>`

Reply:
sounds good, will do!

Reply:
made this dynamic (added `_md` to the existing string)